pricenode/src/main/java/bisq/price/spot/ExchangeRateController.java

```diff
@@ -28,13 +31,29 @@
 class ExchangeRateController extends PriceController {
 
     private final ExchangeRateService exchangeRateService;
+    private final FeeRateService feeRateService;
 
-    public ExchangeRateController(ExchangeRateService exchangeRateService) {
+    public ExchangeRateController(ExchangeRateService exchangeRateService, FeeRateService feeRateService) {
         this.exchangeRateService = exchangeRateService;
+        this.feeRateService = feeRateService;
     }
 
     @GetMapping(path = "/getAllMarketPrices")
     public Map<String, Object> getAllMarketPrices() {
-        return exchangeRateService.getAllMarketPrices();
+        Map<String, Object> retVal = exchangeRateService.getAllMarketPrices();
+
+        // add the fee info to results
+        Map<String, Object> feeInfo = feeRateService.getFees();
+        for (String key : feeInfo.keySet()) {
+            retVal.put(translateFieldName(key), feeInfo.get(key));
+        }
```

Comment:
It's better to either iterate over `feeRateService.getFees().entrySet()` to avoid value lookups for each key and to make your intent clearer or use `feeRateService.getFees().forEach((k, v) -> ...)`